packages/vue/src/components/alias-control.vue

```diff
@@ -20,15 +31,8 @@ const {
   required,
   dialCode,
   dialCodeList,
-} = withDefaults(defineProps<PropsInterface>(), {
-  label: 'Username',
-  name: 'username',
-  placeholder: '',
-  autocomplete: '',
-  labelHidden: false,
-  required: true,
-  type: 'text',
-});
+} = toRefs(props);
```

Comment:
In Vue you lose reactivity if you destructure props, so you have to wrap it in `toRefs` which turns every object into a ref.